src/simulation/sensors/imuSensor/imuSensor.cpp

```diff
@@ -54,10 +54,8 @@ ImuSensor::ImuSensor()
     this->AMatrixGyro.fill(0.0);
```

Comment:
Shouldn't this AmatrixGyro be initialized to a 3x3 identity matrix as you do with the starTracker module?  Here you are leaving it as a zero matrix by default.

Be sure to document default behavior for each module so the user knows what to expect if this Amatrix is not set.